src/index.js

```diff
@@ -7,12 +7,14 @@ addListItem.addEventListener('change', () => {
   const AddedItem = new OneTaskListItem();
   AddedItem.addNewToDo(MyToDo.tasks, addListItem.value);
 });
+// clear completed tasks from list
 const clearSelectedTasks = document.querySelector('.clear');
 clearSelectedTasks.addEventListener('click', () => {
   MyToDo.tasks = MyToDo.tasks.filter((task) => task.completed === false);
   localStorage.setItem('todotasks', JSON.stringify(MyToDo.tasks));
   task.displayToDoList();
 });
```

Comment:
- [ ] [OPTIONAL] Please update the todo `index `keys when you clear all completed tasks 🙏

Reply:
Hi [codepantha](https://github.com/codepantha). Thanks for this suggestion. I will certainly work on it over the weekend. Have a good one.

Reply:
You are welcome, @TracK92. Enjoy your weekend 👋